### ISSUE TITLE
Tensorflow Lite Android - No Operation named [input] in the Graph

### ISSUE THREAD

**Issue by myfknoll** [tensorflow/tensorflow#23534]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Colab
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: Nexus 5X, Samsung S5
- TensorFlow installed from (source or binary):
- TensorFlow version:  1.12.0-rc2
- Python version: 3
- Installed using virtualenv? pip? conda?:  Google Colab
- Bazel version (if compiling from source): Google Colab
- GCC/Compiler version (if compiling from source): Google Colab
- CUDA/cuDNN version: Google Colab
- GPU model and memory: Google Colab

I am building a simple random model with keras, export it as tensorflow lite model from Google Colab, load it on Android, but i am unnable to run classification

My Colab notebook file is available here:
https://colab.research.google.com/drive/1vf9wVXlNQ77GUF_JyL3bKv0M72aK0xvG

The resulting model can be found here
[model.zip](https://github.com/tensorflow/tensorflow/files/2550726/model.zip)

When i open the model with Netron it looks like following:
![model](https://user-images.githubusercontent.com/2804470/48028365-9695da00-e14b-11e8-8ae1-e262510afead.png)

The Android code:
` 
private static final String MODEL_FILE = "file:///android_asset/model.tflite";
private static final String INPUT_NODE = "input";
private static final long[] INPUT_SIZE = {3};
private static final String[] OUTPUT_NODES = {"output/Softmax"};
private static final String OUTPUT_NODE = "output/Softmax";
private static final int OUTPUT_SIZE = 2;

inferenceInterface = new TensorFlowInferenceInterface(inputStream);
float[] data = new float[]{100,200,300};
inferenceInterface.feed(INPUT_NODE, data, INPUT_SIZE);`

But I when i call "feed", I always always an exception:
` java.lang.IllegalArgumentException: No Operation named [input] in the Graph`

I tried different input names, and also specify the input from the converter, but still no success and i am really stucked here. 
Does anyone have an idea what i am actually doing wrong?

**Comment by ZJPUBG** [comment]
Have you tried "input_input" ? In the Netron, the id says it's "input_input".

**Comment by myfknoll** [comment]
Yeah, i tried already a lot. Always same result:
No Operation named [input_input] in the Graph

I may think that the model could be the problem, cause its very small (2kb).. but i don't know..

**Comment by nickaein-a** [comment]
list all operations, it's empty

`Iterator<Operation> operations = inferenceInterface.graph().operations();
        do {
            Operation next = operations.next();
            Log.d(TAG, "op:" + next);
        } while (operations.hasNext());`

i have same problem for my model

**Comment by nickaein-a** [comment]
@shashishekhar you're right
@myfknoll check official [tutorial](https://codelabs.developers.google.com/codelabs/tensorflow-for-poets-2-tflite/#6)

**Comment by yogeshmishrikotkar** [comment]
how you solve it?

**Comment by myfknoll** [comment]
I am using now a different dependency
`implementation 'org.tensorflow:tensorflow-lite:1.10.0'`

You can find this also in the tutorial linked by @nickaein-a above

**Comment by fahadalamkhan3** [comment]
can anyone help me to use the latest version. I think i am lost

**Comment by raminenirama** [comment]
Hi,
Latest version:

implementation 'org.tensorflow:tensorflow-android:1.13.1'

**Comment by GustavoMSevero** [comment]
Hi @myfknoll !
I'm having the same problem, but running in react-native.
Where did you put this line "implementation 'org.tensorflow:tensorflow-lite:1.10.0'"

Thank you

**Comment by ymodak** [comment]
@GustavoMSevero You can try in your app's `build.gradle` file under `dependencies { }`.
